app/cypress/support/index.js

```diff
@@ -1,3 +1,6 @@
 import './commands';
 import 'cypress-axe';
 require('cypress-plugin-retries');
+import authenticatedPages from './authenticatedPages.json';
+
+Cypress.env.AUTHENTICATED_PAGES = authenticatedPages;
```

Comment:
cypress will try importing this file before it's created. See comment below about writing directly to the env variable

Reply:
I had a `precypress` script that ensured the file is created before this import, but this just reminded me that on CI Cypress is run via `test:e2e` instead of the `cypress` script I use locally. So it needed a `pretest:e2e` as well for CI.

```json
"precypress": "node cypress/support/listAuthenticatedPages.js",
```